tests/requirements.txt

```diff
@@ -3,7 +3,8 @@
 anaconda-client
 beautifulsoup4
 chardet
-conda >=4.13
+conda >= 4.13
```

Comment:
```suggestion
conda >=4.13
```
IIRC matchspec parsing can be brittle, was this change intentional?

Reply:
No